pom.xml

```diff
@@ -287,21 +280,32 @@
             <build>
                 <plugins>
                     <plugin>
-                        <groupId>org.sonatype.plugins</groupId>
```

Comment:
We don't need this plugin anymore? It's still in place for REST client: https://github.com/eclipse/microprofile-rest-client/blob/master/pom.xml#L297

Reply:
The properties `autorelease` and `keepStagingReposOnFailure` are left defined but unused if we remove this plugin definition.

Reply:
The sonatype plugin speeds up the process of uploading artifacts to sanatype. Using the default maven dpeloy plugin is very slow.

Reply:
good point. Let me put it back in.